src/Features/Core/Portable/ExternalAccess/VSTypeScript/VSTypeScriptNavigateToSearchService.WrappedNavigableItem.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
+using Microsoft.CodeAnalysis.Navigation;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
+{
+    internal sealed class VSTypeScriptNavigableItem : INavigableItem
```

Comment:
❔ How is this used? It seems like it may be an unnecessary layer of indirection.

Reply:
Type Script gives us `IVSTypeScriptNavigableItem` and we need to store it as `INavigableItem`.

Reply:
We could just have `VSTypeScriptNavigableItem` implement `INavigableItem` and move it to `Api` but Type Script is already using `IVSTypeScriptNavigableItem`. 
I'll add the impl and we can clean up in next update.

Reply:
Will leave it as is for now. There are other issues...